.flake8

```diff
@@ -2,3 +2,7 @@
 per-file-ignores =
    volunteer_planner/settings/*.py: F403, F405,
    volunteer_planner/settings/__init__.py: F403,
+
+max-line-length = 88
+select = C,E,F,W,B,B950
+extend-ignore = E203, E501
```

Comment:
```suggestion
select = C,E,F,W,B,B950
extend-select = B950
extend-ignore = E203, E501
```

Reply:
So if we leave it, it's at least useful for local checks, isn't it?